src/CONST.js

```diff
@@ -70,6 +71,13 @@ const CONST = {
             CHAT: 'chat',
             IOU: 'iou',
         },
+        CHAT_TYPE: {
+            DEFAULT_ROOMS: {
```

Comment:
This is probably an unnecessary layer. I think CHAT_TYPE direct to `POLICY_ANNOUNCE`, `POLICY_ADMINS`, `DOMAIN_ALL` would be fine